validation/terminology-service.rst

```diff
@@ -100,4 +100,54 @@ After processing your request, the terminology server returns the expanded ``Val
     var result = await svc.Expand(parameters) as ValueSet;
 
 
+MultiTerminologyService
+-----------------------
+
+``MultiTerminologyService`` allows you to combine multiple terminology services. This is useful when you have terminology service that are specialized in certain ValueSet or if you want to first check if you first want check if you can process codes locally before consulting an external terminology service.
```

Comment:
"or if you want to first check if you first want check" ->

"or if you want to first check if you can process...."